### PR TITLE
Switch from sqlite to postgresql and migrate on docker up

### PR DESCRIPTION
**Pull request by danielchen115**
Use postgresql instead of sqlite, and migrate on `docker-compose up -d`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ services:
       - POSTGRES_PASSWORD=postgres
   web:
     build: .
-    command: python manage.py runserver 0.0.0.0:8000
+    command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
```
Comment (Tayef-Shah) [inline docker-compose.yml:12]: Does 0.0.0.0 direct to the local IP address?

File: projectread/settings.py [hunk 2]
```diff
@@ -83,12 +83,15 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": "postgres",
+        "USER": "postgres",
+        "PASSWORD": "postgres",
```
Comment (Tayef-Shah) [inline projectread/settings.py:91]: Do we wanna move the password into .env? Also, are we planning to change the password soon?

**Review by Tayef-Shah — COMMENTED** [review]
I was a little curious about the db credentials, because I'm imagining they will be different for production vs local. How do we plan to manage that?

File: projectread/settings.py (hunk 2 above)
Comment (danielchen115) [reply projectread/settings.py:91]: Should be fine :) It's just the development database and it won't (or at least shouldn't) have any sensitive data

**Review by annoyingpr — APPROVED** [review]
🚀   🐘

**Pull request merged**